iam/src/index.ts

```diff
@@ -244,9 +252,12 @@ app.post("/api/v0.0.0/verify", (req: Request, res: Response): void => {
           );
         }
 
-        const result = await checkConditionsAndIssueCredentials(payload, address);
+        const types = payload.types.filter((type) => type);
```

Comment:
- verifyChallengeAndGetAddress, hasValidIssuer - needs to be reusable in `embed`

Reply:
to be handled in: https://github.com/passportxyz/passport/issues/3193